rosgraph_monitor_msgs/msg/Parameter.msg

```diff
@@ -0,0 +1,5 @@
+# Message representing a single ROS2 parameter
+
+string name  # Parameter name
```

Comment:
Descriptor holds the name, so maybe it's just Descriptor+Value?

And we can start with only filling in the `descriptor.name` - leaving `value.type = PARAMETER_NOT_SET` and the rest of the descriptor fields empty. Which puts the data structure in place and just starts filling out the data we know so far.